modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -56,9 +57,21 @@ apiVersion: "k8s.cni.cncf.io/v1"
 ----
 <1> The type is `bond`.
 <2> The `ifname` attribute specifies the name of the bond interface.
-<3> The `failover` attribute is mandatory for active-backup mode.
-<4> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV/Multus.
-<5> The `links` section defines which interfaces will be used to create the bond. By default, Multus names the attached interfaces as: "net", plus a consecutive number, starting with one.
+<3> The `mode` attribute specifies the bonding mode. 
```

Comment:
We still have <4>, <5> and <6> in the example, but the explanation is removed.
Either remove these from example, or keep the explanation